bigquery_etl/query_scheduling/task.py

```diff
@@ -220,6 +220,14 @@ def validate_deploy_type(self, attribute, value):
             )
 
 
+@attr.s(auto_attribs=True, frozen=True)
+class TaskContainerResources:
+    """Represents the Kubernetes container resources configuration for a task."""
+
+    requests: Optional[Dict[str, str]] = attr.ib(None)
+    limits: Optional[Dict[str, str]] = attr.ib(None)
```

Comment:
It would be useful to have an example as a reminder for what these are supposed to look like
```suggestion
    """Represents the Kubernetes container resources configuration for a task."""

    # example dict: {"memory": "4Gi", "cpu": "500m"}
    requests: Optional[Dict[str, str]] = attr.ib(None)
    limits: Optional[Dict[str, str]] = attr.ib(None)
```